src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -351,6 +360,9 @@ public abstract partial class DbDataReader : System.Collections.IEnumerable, Sys
         public abstract bool Read();
         public System.Threading.Tasks.Task<bool> ReadAsync() { return default(System.Threading.Tasks.Task<bool>); }
         public virtual System.Threading.Tasks.Task<bool> ReadAsync(System.Threading.CancellationToken cancellationToken) { return default(System.Threading.Tasks.Task<bool>); }
+        System.Data.IDataReader System.Data.IDataRecord.GetData(int ordinal) { return default(System.Data.IDataReader); }
+        public void Close() { }
+        public DataTable GetSchemaTable() { return default(DataTable); }
```

Comment:
Should `GetSchemaTable` be implemented explicitly, so it doesn't show up in intellisense by default, since `DataTable` in CoreFX is just an empty class?